Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -2347,7 +2347,7 @@ def create_channel():
 def invite_to_channel():
     channel = demisto.args().get('channel')
     channel_id = demisto.args().get('channel_id', '')
-    users = argToList(demisto.args().get('users', []))
+    users = argToList(demisto.args().get('users', '[]').rstrip(', '))
```

Comment:
Why do we want to support a malformed list?

Reply:
:hide-the-pain: Because it's a "friction point"